### PR TITLE
Removing unreleased feature

### PR DESCRIPTION
**Pull request by get-itips**
The option has still no effect

closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8898

Converted to draft to get a final confirmation

### CODE REVIEW

File: Teams/messaging-policies-in-teams.md [hunk 1]
```diff
@@ -75,7 +75,6 @@ Here are the messaging policy settings that you can configure.
 
 - **Owners can delete sent messages**  Use this setting to let owners delete channel messages or posts that users sent.
 - **Delete sent messages** Use this setting to let users delete messages that they sent in chat.
-- **Delete chat** Use this setting to let users delete messages that they sent in chat.
 - **Edit sent messages** Use this setting to let users edit the messages that they sent in chat.
 - **Read receipts** Read receipts allow the sender of a chat message to be notified when their message was read by the recipient in 1:1 and group chats 20 people or fewer. Message read receipts remove uncertainly about whether a message was read, and improve team communication. Read receipts aren't captured in eDiscovery reporting.
```
Comment (JohanFreelancer9) [inline Teams/messaging-policies-in-teams.md:79]:
```suggestion
- **Read receipts** Read receipts allow the sender of a chat message to be notified when their message was read by the recipient in 1:1 and group chats 20 people or fewer. Message read receipts remove uncertainty about whether a message was read, and improve team communication. Read receipts aren't captured in eDiscovery reporting.  
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Hi @get-itips, not related, but could you please also correct the typo in line 79? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@mkbond007 please review

**Comment by get-itips** [comment]
@tiburd 
image

Option has been finally removed from TAC, can you help me merge? Thanks

**Pull request merged**